<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>853</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Titlebar" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-3</x>
     <y>-1</y>
     <width>861</width>
     <height>61</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>816</x>
      <y>14</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>14</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Output as picture</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>335</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>483</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>434</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>384</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="mylineedit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="mycombobox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="mycombobox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>430</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="mylineedit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>480</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>D:/</string>
   </property>
  </widget>
  <widget class="mypushbutton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>582</x>
     <y>480</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="mypushbutton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>689</x>
     <y>480</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Output</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>60</y>
     <width>261</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>261</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>230</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>逐页输出</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>531</x>
     <y>230</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>合成长图</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Titlebar</class>
   <extends>QWidget</extends>
   <header>titlebar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mycombobox</class>
   <extends>QComboBox</extends>
   <header location="global">mycombobox.h</header>
  </customwidget>
  <customwidget>
   <class>mylineedit</class>
   <extends>QLineEdit</extends>
   <header location="global">mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>mypushbutton</class>
   <extends>QPushButton</extends>
   <header location="global">mypushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
